File: files/300-399/391/CH-391.R
library(tidyverse)
library(readxl)

path <- "300-399/391/CH-391 Index.xlsx"
input <- read_excel(path, range = "B3:F23")
test <- read_excel(path, range = "G3:G23")

result = input %>%
  mutate(Mark = ifelse(row_number() == 1, "*", NA_character_), .by = "Customer")

all.equal(result$Mark, test$Mark)
# [1] TRUE

File: files/300-399/391/CH-391.py
import pandas as pd

path = "300-399/391/CH-391 Index.xlsx"
input = pd.read_excel(path, usecols="B:F", nrows=21, skiprows=2)
test = pd.read_excel(path, usecols="G", nrows=21, skiprows=2)

input['Mark'] = input.groupby('Customer').cumcount().apply(lambda x: '*' if x == 0 else None)

print(input['Mark'].equals(test['Mark']))
# [1] True